src/WebJobs.Script.Grpc/Abstractions/IRpcServer.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.Abstractions.Rpc
+{
+    public interface IRpcServer
+    {
+        void Start();
+
+        Task ShutdownAsync();
```

Comment:
When is this Task signalled? After shutdown is fully complete?  If the worker is running functions, do we need to wait for executions to finish?

Reply:
The task will signal after shutdown is fully complete. 

I haven't tested what Grpc does if we start after shutdown, but I think that semantics of 'need new instance after shutdown' will make different implementations of this more flexible.